csrc/ops.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <optional>
```

Comment:
Why is this needed?

Reply:
Without this I get these errors:

```
In file included from /workspace/vllm/csrc/cpu/torch_bindings.cpp:2:
/workspace/vllm/csrc/ops.h:58:36: error: ‘optional’ in namespace ‘std’ does not name a template type
   58 |                         const std::optional<torch::Tensor>& bias);
      |                                    ^~~~~~~~
/workspace/vllm/csrc/ops.h:1:1: note: ‘std::optional’ is defined in header ‘<optional>’; did you forget to ‘#include <optional>’?
  +++ |+#include <optional>
```

Reply:
Yes, I believe c++17 is enabled by default when using the CUDA backend because it is required for CUTLASS/modern CUDA. This is fine then